backend/src/game/newcards/new_card.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC
+
+
+class NewCard(ABC):
```

Comment:
I am not in love with this class name here but I do not currently have an alternative.  Maybe just GameplayCard or something more to that effect

Reply:
It will be Card, it's temporarily NewCard while we haven't retired Chris' parser code yet.